src/components/interfaces/HMI_API.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" standalone="yes"?>
 
 <!--
-* Copyright (c) 2016, Ford Motor Company
+ * Copyright (c) 2016, Ford Motor Company
```

Comment:
@masatoogawa 2017

Reply:
The first year 2016 will disappear. Is it a policy of copyright notation of this project?

Reply:
@LuxoftAKutsan Honestly, this should be updated as part of any contribution from Ford this year, but it never was. The copyright likely needs to be updated in several files, but that can be done in a separate PR. @masatoogawa I wouldn't worry about it for this PR.